app/models/sbv/person.rb

```diff
@@ -19,4 +19,10 @@ module Sbv::Person
 
   end
 
+  def active_years
```

Comment:
Wenn ein Mitglied in mehreren Vereinen gleichzeitig tätig ist, werden meiner Ansicht nach zu viele Jahre verrechnet. Bei einer Tätigkeit in zwei Vereinen gleichzeitig für 4 Jahre, würde dies schlussendlich 8 Aktivjahre ergeben, aber es sollten allerdings nur 4 sein.

Reply:
Interessanter Punkt. `sum` verwendet das `+` aus dem Domain-Objekt, welches nur Start und Ende erweitert. Daher werden es nicht zuviele Jahre. Es könnten jedoch zuwenige werden, da die "passive years" stumpf zusammengenommen werden. Wenn man also einen Unterbruch in einem Verein an, weil man in der Zeit in einem anderen Verein ist, könnten eher Jahre fehlen, weil die in einen Jahre die anderen überlagern. Mit einem `sort` sollte das Problem aber umgangen werden, weil dann die Zeiträume chronologisch aneinandergereiht werden. Ich schaue mir das noch an.

Reply:
Achsoo, das sum verwendet dein erstelltes +. Das habe ich ganz übersehen, aber clever gelöst. In dem Fall ist meine Befürchtung verflogen und das mit dem sort könnte so funktionieren.